compiler/rustc_ast/src/ast.rs

```diff
@@ -1125,8 +1125,8 @@ pub struct Expr {
 }
 
 // `Expr` is used a lot. Make sure it doesn't unintentionally get bigger.
-#[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
-rustc_data_structures::static_assert_size!(Expr, 104);
+//#[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
```

Comment:
You can use `bootstrap` in the cfg feature list (or `not(bootstrap)` to be precise)